portal/v2/src/ClusterDetailListComponents/Statistics/GraphOptionsComponent.tsx

```diff
@@ -0,0 +1,220 @@
+        marginLeft: 5,
+      },
+    })
+    const classNames = mergeStyleSets({
+      iconContainer: {
+        margin: "0px 0px",
+        height: 25,
+        width: "90px",
+      },
+    })
+  
+    const datePickerStyles: Partial<IDatePickerStyles> = { root: { maxWidth: 145, marginTop: 5, } };
+    const timePickerStyles: Partial<IDatePickerStyles> = { root: { maxWidth: 75, marginLeft: 5, marginRight: 5 } };
+    const onTimeChange = (event: React.FormEvent<IComboBox>, date: Date) => {        
+      var localDate = new Date()
+      localDate.setUTCFullYear(props.endDate.getFullYear(), props.endDate.getMonth(), props.endDate.getDate())
+      localDate.setUTCHours(date.getHours());
+      localDate.setUTCMinutes(date.getMinutes());
+      props.setEndDate(localDate)        
+     };
+    const onDateChange = (date: Date | null | undefined): void => { 
+      let localDate: Date = new Date();
+      localDate.setUTCFullYear(date!.getFullYear(), date!.getMonth(), date!.getDate())
+      localDate.setHours(props.endDate.getHours());
+      localDate.setMinutes(props.endDate.getMinutes());
+      props.setEndDate(localDate)  
+     };
+     
+    return (
+      <>
+      <Stack horizontal verticalAlign="center">
+      <Stack horizontal verticalAlign="center" className={classNames.iconContainer} /* style={{ boxShadow: theme.effects.elevation8 }}*/>
+          <TooltipHost>
+            <IconButton
+              onClick={_decreaseDuration}
+              iconProps={calculatorSubtract}
+              styles={iconButtonStyles}
+            />
+          </TooltipHost>
+          <TooltipHost>
+            <Text styles={durationStyle}>{props.duration}</Text>
+          </TooltipHost>
+          <TooltipHost >
+            <IconButton
+              onClick={_increaseDuration}
+              iconProps={calculatorAddition}
+              styles={iconButtonStyles}
+            />
+          </TooltipHost>
+        </Stack>
+        <Stack horizontal verticalAlign="center" className={dateTimePickerStyles.iconContainer}  /* style={{ boxShadow: theme.effects.elevation8 }}*/>
+          <DatePicker
+            styles={datePickerStyles}
+            placeholder="End Date"
+            ariaLabel="End Date"
+            onSelectDate={onDateChange}
+            value={convertToUTC(props.endDate)}
+            allowTextInput
+          />
+          <TimePicker
+            styles={timePickerStyles}
+            allowFreeform
+            placeholder={timeToString(convertToUTC(props.endDate))}
+            autoComplete="on"
+            onChange={onTimeChange}
+            defaultValue={convertToUTC(props.endDate)}
+            useComboBoxAsMenuWidth
+          />
+        </Stack>
+      </Stack>
+      </>
+    )
+  }
+
+  function timeToString(date: Date): string {
+    var str
+    let hourString = date.getHours().toString()
+    if (hourString.length === 1) {
+      str = "0" + hourString + ":"
+    } else {
+      str = hourString + ":"
+    }
```

Comment:
No need for the else. Just set `str = ...` above the `if`.

Reply:
yup done.